sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_web_referrer_v1/query.sql

```diff
@@ -0,0 +1,104 @@
+WITH views_territory AS (
+  SELECT
+    DATE(web_referrer.`date`) AS `date`,
+    web_referrer.app_id,
+    web_referrer.territory,
+    web_referrer.web_referrer,
+    COALESCE(campaign.campaign, "MISSING") AS campaign,
```

Comment:
To make sure the join at the very end works correctly, afterwards in the final select this value is nulled.